test/core/settings.test.js

```diff
@@ -24,7 +24,7 @@ describe('Core.Settings', function () {
 	});
 
 	it("should work between different require'd files.", function () {
-		assert.equal(require("../../src/javascript/core/settings").get('key2'), "value2");
+		assert.equal(Settings.get('key2'), "value2");
```

Comment:
This test is incorrect now because it is not testing between different `require`'d calls.